src/styles/app.scss

```diff
@@ -761,19 +761,22 @@ $border-colour: govuk-colour("grey-2");
     }
 }
 
-.govuk-form-group {
-  .govuk-fieldset {
-    .govuk-table {
-      table-layout: fixed;
-    }
-  }
-}
+#Correspondents, #DraftDocuments, #Topics {
+  table {
+    table-layout: fixed;
+    overflow-wrap: break-word;
+
+    > tbody > tr > td {
```

Comment:
Would just `td` be sufficient? (And more efficient)

Reply:
Verified #Correspondents, #DraftDocuments, #Topics against the form data (loaded in hocs-data project).
It all looks good to me. The govuk-form-group and govuk-table__cell are placed here by the new formatting. 
There is no danger of affecting other components or formatting as they were both inserted in the css for this ticket.